package.json

```diff
@@ -33,6 +33,7 @@
   "devDependencies": {
     "all-contributors-cli": "^6.15.0",
     "npm-run-all": "^4.1.5",
+    "nyc": "^15.1.0",
```

Comment:
I can't see any usage, do we really need this?

Reply:
It's a mistake I remove this